### DISCUSSION TITLE
Consider to add better movement for chinese?

### DISCUSSION THREAD

**Discussion by LeoGitHuber** [meow-edit/meow#499] [General]
Add chinese support like https://github.com/kanglmf/emacs-chinese-word-segmentation for better movement and mark.

**Comment by DogLooksGood** [comment]
No, there's no such plan. Meow is just a modal layer, the behaviors about `word` is completely decided by how `word` thing is defined.
Check [thingatpt.el](https://github.com/emacs-mirror/emacs/blob/master/lisp/thingatpt.el#L27).

To support Chinese word segmentation, you have to override the `word` thing definition. It affects all word related commands in Emacs, including Meow's word commands.
